server/lobbyconnection.py

```diff
@@ -216,6 +216,15 @@ async def on_message_received(self, message):
                 self.get_user_identifier(),
                 cmd
             )
+        except OperationalError:
+            # When the database goes down, SqlAlchemy will throw an OperationalError
```

Comment:
I can go ahead and add logging here. I was avoiding it only because if the DB is truly down, it could cause a lot of logs to the logging service to be ingested due to failed commands. But if we feel like the logging service can handle that volume without issue, I have no qualms about adding it in.

Reply:
Too easy. I've added that logging in the most recent version.